include/lsst/afw/geom/Endpoint.h

```diff
@@ -179,13 +179,7 @@ class BaseVectorEndpoint : public BaseEndpoint<PointT, std::vector<PointT>> {
 
     virtual ~BaseVectorEndpoint(){};
 
-    virtual int getNPoints(Array const &arr) const override { return arr.size(); }
-
-    virtual std::vector<double> dataFromPoint(Point const &point) const override;
-
-    virtual ndarray::Array<double, 2, 2> dataFromArray(Array const &arr) const override;
-
-    virtual Point pointFromData(std::vector<double> const &data) const override;
+    virtual int getNPoints(Array const &arr) const override;
```

Comment:
Because for `GenericEndpoint.getNPoints` Array is a 2-dimensional ndarray and so can't use `size`